File: app/src/main/java/ru/systempla/weatherapp/navigation/Screens.java
package ru.systempla.weatherapp.navigation;

import androidx.fragment.app.Fragment;

import ru.systempla.weatherapp.mvp.view.ui.fragment.ForecastFragment;
import ru.systempla.weatherapp.mvp.view.ui.fragment.WeatherDataFragment;

import ru.terrakok.cicerone.android.support.SupportAppScreen;

public class Screens {
    public static class WeatherDataScreen extends SupportAppScreen {
        @Override
        public Fragment getFragment() {
            return WeatherDataFragment.newInstance();
        }
    }

    public static class WeekForcastScreen extends SupportAppScreen {
        @Override
        public Fragment getFragment() {
            return ForecastFragment.newInstance();
        }
    }
}

File: app/src/main/java/ru/systempla/weatherapp/navigation/Screens.kt
package ru.systempla.weatherapp.navigation

import androidx.fragment.app.Fragment
import ru.systempla.weatherapp.mvp.view.ui.fragment.ForecastFragment
import ru.systempla.weatherapp.mvp.view.ui.fragment.WeatherDataFragment
import ru.terrakok.cicerone.android.support.SupportAppScreen

class Screens {
    class WeatherDataScreen : SupportAppScreen() {
        override fun getFragment(): Fragment {
            return WeatherDataFragment.newInstance()
        }
    }

    class WeekForecastScreen : SupportAppScreen() {
        override fun getFragment(): Fragment {
            return ForecastFragment.newInstance()
        }
    }
}